capstone/client/src/components/StoreMaps.js

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2020 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { GoogleApiWrapper, Map, Marker } from 'google-maps-react';
+import React, { Component } from 'react';
+import Geocode from 'react-geocode';
+
+import APIKey from './APIKey.js';
+import { StoresContext } from './StoresProvider.js';
+import './styles.css';
+
+class StoreMaps extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      latitude: 0,
+      longitude: 0
+    }
+  } 
+
+  getLatLangFromAddress = (address) => {
+    Geocode.fromAddress(address).then((res) => {
+      const {lat, lng} = res.results[0].geometry.location;
+      this.setState({
+          latitude: lat,
+          longitude: lng
+      });
+    }).catch((error) => {
+      this.setState({
+        errorMessage: "There was an error showing store locations.",
+      })
+    });
+  }
+
+  componentDidMount = () => {
+    Geocode.setApiKey(APIKey.GeocodeAPIKey());
```

Comment:
change your API key stored locally to your GeocodeAPIKey so you only use one function to get your key, there should be one universal key you can use so you only need one function APIKey()

this wont work on any of our stuff locally

Reply:
When I used the API key that we use for Maps (APIKey.APIKey()), I got an error because that API key is only enabled for Maps API (nothing changed when I added the Geocoding API to the non-restricted APIs, so I don't know what happened there). How did you work it out with your API key?

Reply:
My API key is working for Geocoding, which key are you using? I think I am using #2

Reply:
Okay, I changed the API key.

Reply:
We might _want_ to have restrictions on our API keys though?